app/cafe.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"Visitor {visitor.get("name", "Unknown")} is not vaccinated."
```

Comment:
The use of double quotes inside the formatted string is incorrect and will cause a syntax error. You should use single quotes around the string or escape the double quotes inside the formatted string.